pkg/kn/commands/revision_describe.go

```diff
@@ -51,10 +51,7 @@ func NewRevisionDescribeCommand(p *KnParams) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			revision.GetObjectKind().SetGroupVersionKind(schema.GroupVersionKind{
-				Group:   "knative.dev",
-				Version: "v1alpha1",
-				Kind:    "Revision"})
+			printer = printers.NewGvkUpdatePrinter(printer)
```

Comment:
In #134 you would call out to `serving.UpdateGroupVersionKind()` instead of wrapping the printer, which calls this method on print. Still, you have a specialized handling.

Reply:
can we hide `printers.NewGvkUpdatePrinter(printer)` in revisionDescribePrintFlags.ToPrinter()?

Reply:
sure, let me align it with the service handling.